docs/feature_layers.md

```diff
@@ -19,12 +19,10 @@ These functions allow you to activate layers in various ways. Note that layers a
 
 ### Caveats :id=caveats
 
-Currently, `LT()` and `MT()` are limited to the [Basic Keycode set](keycodes_basic.md), meaning you can't use keycodes like `LCTL()`, `KC_TILD`, or anything greater than `0xFF`. Specifically, dual function keys like `LT` and `MT` use a 16 bit keycode. 4 bits are used for the function identifier, the next 12 are divided into the parameters. Layer Tap uses 4 bits for the layer (and is why it's limited to layers 0-15, actually), while Mod Tap does the same, 4 bits for the identifier, 4 bits for which mods are used, and all of them use 8 bits for the keycode. Because of this, the keycode used is limited to `0xFF` (0-255), which are the basic keycodes only. 
+Currently, `LT()` (and also all [Mod-Tap](mod_tap.md) keycodes) are limited to the [Basic Keycode set](keycodes_basic.md) for their *kc* argument, meaning you can't use keycodes like `LCTL()`, `KC_TILD`, or anything greater than `0xFF`. In the case of `LT()`, this is because QMK uses 16-bit keycodes, of which 4 bits are used for the function identifier, 4 bits for the layer (and this is why it's limited to layers 0-15, actually), only 8 bits are left for the keycode. Because of this, the keycode used is limited to `0xFF` (0-255), which are the basic keycodes only. 
```

Comment:
```suggestion
Currently, the `layer` argument of `LT()` is limited to layers 0-15, and the `kc` argument to the [Basic Keycode set](keycodes_basic.md), meaning you can't use keycodes like `LCTL()`, `KC_TILD`, or anything greater than `0xFF`. This is because QMK uses 16-bit keycodes, of which 4 bits are used for the function identifier and 4 bits for the layer, leaving only 8 bits for the keycode.
```